[2025-10-29T13:42:01.208+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag1_calcul_trajet_kafka.ConsumKafka scheduled__2025-10-29T13:41:00+00:00 [queued]>
[2025-10-29T13:42:01.217+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag1_calcul_trajet_kafka.ConsumKafka scheduled__2025-10-29T13:41:00+00:00 [queued]>
[2025-10-29T13:42:01.218+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-10-29T13:42:01.233+0000] {taskinstance.py:2191} INFO - Executing <Task(_PythonDecoratedOperator): ConsumKafka> on 2025-10-29 13:41:00+00:00
[2025-10-29T13:42:01.241+0000] {standard_task_runner.py:60} INFO - Started process 457 to run task
[2025-10-29T13:42:01.246+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dag1_calcul_trajet_kafka', 'ConsumKafka', 'scheduled__2025-10-29T13:41:00+00:00', '--job-id', '71', '--raw', '--subdir', 'DAGS_FOLDER/dag1_kafka_processing.py', '--cfg-path', '/tmp/tmpwc5e1k2i']
[2025-10-29T13:42:01.248+0000] {standard_task_runner.py:88} INFO - Job 71: Subtask ConsumKafka
[2025-10-29T13:42:01.310+0000] {task_command.py:423} INFO - Running <TaskInstance: dag1_calcul_trajet_kafka.ConsumKafka scheduled__2025-10-29T13:41:00+00:00 [running]> on host c91e5c2c38a7
[2025-10-29T13:42:01.407+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag1_calcul_trajet_kafka' AIRFLOW_CTX_TASK_ID='ConsumKafka' AIRFLOW_CTX_EXECUTION_DATE='2025-10-29T13:41:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-10-29T13:41:00+00:00'
[2025-10-29T13:42:01.413+0000] {conn.py:396} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=bootstrap-0 host=kafka:9092 <connecting> [IPv4 ('172.18.0.6', 9092)]>: connecting to kafka:9092 [('172.18.0.6', 9092) IPv4]
[2025-10-29T13:42:01.416+0000] {conn.py:618} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=bootstrap-0 host=kafka:9092 <checking_api_versions_recv> [IPv4 ('172.18.0.6', 9092)]>: Broker version identified as 2.6
[2025-10-29T13:42:01.417+0000] {conn.py:457} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=bootstrap-0 host=kafka:9092 <connected> [IPv4 ('172.18.0.6', 9092)]>: Connection complete.
[2025-10-29T13:42:01.419+0000] {subscription_state.py:181} INFO - Updating subscribed topics to: ('source',)
[2025-10-29T13:42:01.420+0000] {dag1_kafka_processing.py:49} INFO - âœ… Consommateur Kafka crÃ©Ã© pour le topic 'source'. En attente de messages...
[2025-10-29T13:42:01.424+0000] {conn.py:396} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=1 host=kafka:9092 <connecting> [IPv4 ('172.18.0.6', 9092)]>: connecting to kafka:9092 [('172.18.0.6', 9092) IPv4]
[2025-10-29T13:42:01.424+0000] {conn.py:457} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=1 host=kafka:9092 <connected> [IPv4 ('172.18.0.6', 9092)]>: Connection complete.
[2025-10-29T13:42:01.425+0000] {conn.py:945} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=bootstrap-0 host=kafka:9092 <connected> [IPv4 ('172.18.0.6', 9092)]>: Closing connection. 
[2025-10-29T13:42:01.528+0000] {cluster.py:401} INFO - Coordinator for group/***-consumers-dag1 is ('coordinator-1', 'kafka', 9092, None)
[2025-10-29T13:42:01.529+0000] {base.py:788} INFO - Discovered coordinator coordinator-1 for group ***-consumers-dag1
[2025-10-29T13:42:01.529+0000] {base.py:848} INFO - Starting new heartbeat thread
[2025-10-29T13:42:01.531+0000] {consumer.py:365} INFO - Revoking previously assigned partitions () for group ***-consumers-dag1
[2025-10-29T13:42:01.531+0000] {base.py:419} INFO - Failed to join group ***-consumers-dag1: NodeNotReadyError: coordinator-1
[2025-10-29T13:42:01.533+0000] {conn.py:396} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=coordinator-1 host=kafka:9092 <connecting> [IPv4 ('172.18.0.6', 9092)]>: connecting to kafka:9092 [('172.18.0.6', 9092) IPv4]
[2025-10-29T13:42:01.534+0000] {conn.py:457} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=coordinator-1 host=kafka:9092 <connected> [IPv4 ('172.18.0.6', 9092)]>: Connection complete.
[2025-10-29T13:42:01.635+0000] {base.py:532} INFO - (Re-)joining group ***-consumers-dag1
[2025-10-29T13:42:01.639+0000] {base.py:637} INFO - Received member id kafka-python-2.2.15-29dbd78c-ba29-48ff-93eb-4c4fd2560a22 for group ***-consumers-dag1; will retry join-group
[2025-10-29T13:42:01.639+0000] {base.py:419} INFO - Failed to join group ***-consumers-dag1: [Error 79] MemberIdRequiredError
[2025-10-29T13:42:01.640+0000] {base.py:532} INFO - (Re-)joining group ***-consumers-dag1
[2025-10-29T13:42:04.646+0000] {base.py:594} INFO - Successfully joined group ***-consumers-dag1 <Generation 34 (member_id: kafka-python-2.2.15-29dbd78c-ba29-48ff-93eb-4c4fd2560a22, protocol: range)>
[2025-10-29T13:42:04.647+0000] {base.py:596} INFO - Elected group leader -- performing partition assignments using range
[2025-10-29T13:42:04.651+0000] {subscription_state.py:254} INFO - Updated partition assignment: [('source', 0)]
[2025-10-29T13:42:04.652+0000] {consumer.py:249} INFO - Setting newly assigned partitions (('source', 0),) for group ***-consumers-dag1
[2025-10-29T13:42:04.659+0000] {dag1_kafka_processing.py:60} INFO - ðŸ“© Message reÃ§u : {'confort': 'medium', 'prix_base_per_km': 2, 'properties-client': {'logitude': 2.4013757323896954, 'latitude': 48.86598603753334, 'nomclient': 'FALL', 'telephoneClient': '060786575'}, 'properties-driver': {'logitude': 3.729108444675927, 'latitude': 40.41121983054842, 'nomDriver': 'DIOP', 'telephoneDriver': '0760786575'}}
[2025-10-29T13:42:04.663+0000] {base.py:1052} INFO - Stopping heartbeat thread
[2025-10-29T13:42:04.664+0000] {base.py:891} INFO - Leaving consumer group (***-consumers-dag1).
[2025-10-29T13:42:04.667+0000] {base.py:906} INFO - LeaveGroup request for group ***-consumers-dag1 returned successfully
[2025-10-29T13:42:04.668+0000] {conn.py:945} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=1 host=kafka:9092 <connected> [IPv4 ('172.18.0.6', 9092)]>: Closing connection. 
[2025-10-29T13:42:04.669+0000] {conn.py:945} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=coordinator-1 host=kafka:9092 <connected> [IPv4 ('172.18.0.6', 9092)]>: Closing connection. 
[2025-10-29T13:42:04.669+0000] {python.py:201} INFO - Done. Returned value was: {'confort': 'medium', 'prix_base_per_km': 2, 'properties-client': {'logitude': 2.4013757323896954, 'latitude': 48.86598603753334, 'nomclient': 'FALL', 'telephoneClient': '060786575'}, 'properties-driver': {'logitude': 3.729108444675927, 'latitude': 40.41121983054842, 'nomDriver': 'DIOP', 'telephoneDriver': '0760786575'}}
[2025-10-29T13:42:04.697+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=dag1_calcul_trajet_kafka, task_id=ConsumKafka, execution_date=20251029T134100, start_date=20251029T134201, end_date=20251029T134204
[2025-10-29T13:42:04.755+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-10-29T13:42:04.784+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
